internal/jwx/jwk/interface.go

```diff
@@ -48,6 +48,7 @@ type RSAPublicKey struct {
 // RSAPrivateKey is a type of JWK generated from RSA private keys
 type RSAPrivateKey struct {
 	*StandardHeaders
+	*jwa.AlgorithmParameters
 	key *rsa.PrivateKey
```

Comment:
I think this may have broken a test over here: https://github.com/open-policy-agent/opa/blob/4747e22f227985a1a04f21da55783469e42f037b/internal/jwx/jwk/rsa_test.go#L44-L46 👉 https://github.com/open-policy-agent/opa/runs/3601600664?check_suite_focus=true#step:3:207

Reply:
Thank you catching it, and I have pushed the fix.